README.md

```diff
@@ -970,68 +970,68 @@ See [go-hardware](https://github.com/rakyll/go-hardware) for a comprehensive lis
-    * [httpexpect](https://github.com/gavv/httpexpect) - Concise, declarative, and easy to use end-to-end HTTP and REST API testing
+    * [httpexpect](https://github.com/gavv/httpexpect) - Concise, declarative, and easy to use end-to-end HTTP and REST API testing.
     * [restit](https://github.com/yookoala/restit) - Go micro framework to help writing RESTful API integration test.
     * [testfixtures](https://github.com/go-testfixtures/testfixtures) - A helper for Rails' like test fixtures to test database applications.
     * [Testify](https://github.com/stretchr/testify) - Sacred extension to the standard go testing package.
     * [wstest](https://github.com/posener/wstest) - Websocket client for unit-testing a websocket http.Handler.
 
 * Mock
-    * [counterfeiter](https://github.com/maxbrunsfeld/counterfeiter) - Tool for generating self-contained mock objects
-    * [go-sqlmock](https://github.com/DATA-DOG/go-sqlmock) - Mock SQL driver for testing database interactions
+    * [counterfeiter](https://github.com/maxbrunsfeld/counterfeiter) - Tool for generating self-contained mock objects.
+    * [go-sqlmock](https://github.com/DATA-DOG/go-sqlmock) - Mock SQL driver for testing database interactions.
     * [go-txdb](https://github.com/DATA-DOG/go-txdb) - Single transaction based database driver mainly for testing purposes.
     * [gock](https://github.com/h2non/gock) - Versatile HTTP mocking made easy.
     * [gomock](https://github.com/golang/mock) - Mocking framework for the Go programming language.
-    * [govcr](https://github.com/seborama/govcr) -  HTTP mock for Golang: record and replay HTTP interactions for offline testing
-    * [minimock](https://github.com/gojuno/minimock) - Mock generator for Go interfaces
-    * [mockhttp](https://github.com/tv42/mockhttp) - Mock object for Go http.ResponseWriter
+    * [govcr](https://github.com/seborama/govcr) -  HTTP mock for Golang: record and replay HTTP interactions for offline testing.
+    * [minimock](https://github.com/gojuno/minimock) - Mock generator for Go interfaces.
+    * [mockhttp](https://github.com/tv42/mockhttp) - Mock object for Go http.ResponseWriter.
 
-* Fuzzing and delta-debugging/reducing/shrinking
-    * [go-fuzz](https://github.com/dvyukov/go-fuzz) - Randomized testing system
-    * [gofuzz](https://github.com/google/gofuzz) - Library for populating go objects with random values
-    * [Tavor](https://github.com/zimmski/tavor) - Generic fuzzing and delta-debugging framework
+* Fuzzing and delta-debugging/reducing/shrinking.
+    * [go-fuzz](https://github.com/dvyukov/go-fuzz) - Randomized testing system.
+    * [gofuzz](https://github.com/google/gofuzz) - Library for populating go objects with random values.
+    * [Tavor](https://github.com/zimmski/tavor) - Generic fuzzing and delta-debugging framework.
 
-* Selenium and browser control tools
+* Selenium and browser control tools.
```

Comment:
This looks like a title, not a sentence. No period needed. Maybe make the titles use capital letters to make it clear:

> * Selenium and Browser Control Tools